scipy/optimize/_linprog.py

```diff
@@ -338,13 +344,30 @@ def linprog(c, A_ub=None, b_ub=None, A_eq=None, b_eq=None,
     This section describes the available solvers that can be selected by the
     'method' parameter.
 
+    :ref:`'highs-simplex' <optimize.linprog-highs-simplex>` and
+    :ref:`'highs-ipm' <optimize.linprog-highs-ipm>` are interfaces to the
+    HiGHS simplex and interior-point method solvers [13]_, respectively.
+    'highs' chooses between the two automatically. These are the fastest linear
+    programming solvers in SciPy, especially for large, sparse problems.
     :ref:`'interior-point' <optimize.linprog-interior-point>` is the default
-    as it is typically the fastest and most robust method.
+    as it was the fastest and most robust method before the recent
+    addition of the HiGHS solvers. This may change in a future release.
     :ref:`'revised simplex' <optimize.linprog-revised_simplex>` is more
-    accurate for the problems it solves.
+    accurate than interior-point for the problems it solves.
     :ref:`'simplex' <optimize.linprog-simplex>` is the legacy method and is
     included for backwards compatibility and educational purposes.
 
+    Method *highs-simplex* is a wrapper of the C++ high performance dual
+    revised simplex implementation (HSOL) [13]_, [14]_. Method *highs-ipm*
+    is a wrapper of a C++ implementation of an **i**\ nterior-\ **p**\ oint
+    **m**\ ethod [13]_; it features a crossover routine, so it is as accurate as
+    a simplex solver. These are the fastest methods available in SciPy,
```

Comment:
This sentence is duplicate, see paragraph above - suggest to remove one of the two

Reply:
Done.